test/unit/test-amp-img.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {AmpImg, installImg} from '../../builtins/amp-img';
+import {AmpImg, installImg} from '../../builtins/amp-img/amp-img';
```

Comment:
Let's just move it to `amp-img/index.js` and then imports don't have to change?

Reply:
I considered this. Every single amp component in the entire repo has the filename `amp-component.js`. If you feel strongly I'm ok with breaking that pattern here.

Reply:
Hmm that's a very good point, although they live in 0.1 etc directory so it doesn't look as weird on import. I would be fine with breaking that pattern here since they're not extensions as much as they are part of the runtime, but I also don't feel strongly. Curious what others think. What's the motivation behind splitting the directories up here?

Reply:
Right now there are only 2 files per component (.md and .js) so lumping all in the same dir wasn't too bad.

I plan on introducing a third, in which case it starts becoming a bit disorganized

Reply:
I'm quite split between the two points. Could we either poll the group at stand-up or pull in a third opinion?

Reply:
poll taken, `amp-img/amp-img` wins. Also very ok with switching to index.js on a later date :)